src/invidious/views/components/description_chapters_widget.ecr

```diff
@@ -0,0 +1,34 @@
+<% if !chapters.empty? %>
+    <div class="description-chapters-section">
+        <hr class="description-content-separator"/>
+        <h4><%=HTML.escape(translate(locale, "video_chapters_label"))%></h4>
+
+        <% if video.automatically_generated_chapters? %> 
+          <h5><%=HTML.escape(translate(locale, "video_chapters_auto_generated_label"))%> </h5>
+        <% end %>
+
+        <div class="description-chapters-content-container">
+            <% chapters.each do | chapter | %>
+                <%- start_in_seconds = chapter.start_ms.total_seconds.to_i %>
+                <a href="/watch?v=<%-= video.id %>&t=<%=start_in_seconds %>">
+                    <div class="chapter" data-onclick="jump_to_time" data-jump-time="<%=start_in_seconds%>">
+                        <div class="thumbnail">
+                            <%- if !env.get("preferences").as(Preferences).thin_mode -%>
+                                <img loading="lazy" class="thumbnail" src="<%-=URI.parse(chapter.thumbnails[-1]["url"].to_s).request_target %>" alt="" data-onclick="jump_to_time" data-jump-time="<%=start_in_seconds%>"/>
+                            <%- else -%>
+                                <div class="thumbnail-placeholder" data-onclick="jump_to_time" data-jump-time="<%=start_in_seconds%>"></div>
+                            <%- end -%>
+                        </div>
+                        <%- if start_in_seconds > 0 -%>
+                            <p data-onclick="jump_to_time" data-jump-time="<%=start_in_seconds%>"><%-= recode_length_seconds(start_in_seconds) -%></p>
+                        <%- else -%>
+                            <p data-onclick="jump_to_time" data-jump-time="<%=start_in_seconds%>">0:00</p>
+                        <%- end -%>
```

Comment:
Can you instead change `recode_length_seconds()` to return `"0:00"` is the provided time is `0`? It also makes sense in all the other places that use that function.

Reply:
I'm a bit hesitant to change that since Invidious also uses a 0 to represent a nil value when length information isn't available